Specs/Scene/ViewportQuadSpec.js

```diff
@@ -77,42 +63,40 @@ defineSuite([
             new Color(1.0, 1.0, 1.0, 1.0));
     });
 
-    it('throws when rendered with without a rectangle', function() {
+    it('throws when rendered without a rectangle', function() {
         viewportQuad.rectangle = undefined;
 
         expect(function() {
-            render(context, frameState, viewportQuad);
+            scene.primitives.add(viewportQuad);
```

Comment:
Only put the part that we expect to throw in the callback so, here and below, move this above:
```javascript
scene.primitives.add(viewportQuad);
```